src/NServiceBus.Core/Scheduling/Schedule.cs

```diff
@@ -9,7 +9,7 @@ namespace NServiceBus
     /// <summary>
     /// Extends the context with scheduling capabilities.
     /// </summary>
-    public static class ScheduleBusExtensions
+    public static class ScheduleExtensions
```

Comment:
since it's on the same namespace as `IBusSession` and it only contains extension methods, we should be fine? Otherwise I'll revert this and limit the PR to the `BusConfigExtensions`.